### ISSUE TITLE
Multiple LoadBalancer services not correct in LB services list

### ISSUE THREAD

**Issue by mscrocdile** [hetznercloud/hcloud-cloud-controller-manager#613]
### TL;DR

There is only one (latest) service port(s) instead of all ports of all annotated services in Hetzner Load Balancer administration.

### Expected behavior

I expect there will be all ports and services in https://console.hetzner.cloud/projects/YOUR/loadbalancers/YOUR/services

### Observed behavior

There is only latest annotated service port(s)

### Minimal working example

If i do this in microk8s cluster:

k -n traefik annotate services traefik  \
  load-balancer.hetzner.cloud/name="mylb" \
  load-balancer.hetzner.cloud/location="myloc" \
  load-balancer.hetzner.cloud/use-private-ip="false" \
  load-balancer.hetzner.cloud/uses-proxyprotocol="true" \
  load-balancer.hetzner.cloud/hostname="mydomain"

then there is 80 and 443 in services and everything works.

and then i do :

k -n mssql annotate services  mymssql \
  load-balancer.hetzner.cloud/name="mylb" \
  load-balancer.hetzner.cloud/location="myloc" \
  load-balancer.hetzner.cloud/use-private-ip="false" \
  load-balancer.hetzner.cloud/uses-proxyprotocol="true" \
  load-balancer.hetzner.cloud/hostname="mydomain"

In k get services --all-namespaces there is everything correct and both services has public ip assigned.

BUT only mssql with port 1433 is there in web LB administration (80 and 443 dissappear) so traefik stops working.

**Comment by mscrocdile** [comment]
Hello, i did not know it.  Then it is ok. Thanks
